src/ServicePulse.Host/app/css/particular.css

```diff
@@ -2943,4 +2943,20 @@ table[ng-show="vm.message.panel === 2 && vm.message.messageHeaders"] td[nowrap="
 
 span.label {
     background-image: none;
+}
+
+a[ng-click="deleteEndpoint(endpoint)"] {
```

Comment:
I would actually feel a lot more comfortable if these selectors were more explicit. By using the `ng-click` attribute, it is possible a future refactoring could change the name of the function and unintentionally change the look of the elements.